packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.js

```diff
@@ -176,26 +223,40 @@ const ErrorFormTooltip = ({
         onShown(false);
       }
     };
-
     window.addEventListener("keydown", handleTab);
     return () => {
       window.removeEventListener("keydown", handleTab);
     };
-  }, [onShown, isHelp]);
+  }, [onShown, isHelp, shown, tooltipRef, update]);
```

Comment:
If you move the call to `update()` in the new `React.useEffect` I suggested above, adding `shown` to dependencies there, you can remove `shown` from here: